client/my-sites/marketplace/pages/marketplace-plugin-install/index.tsx

```diff
@@ -395,8 +394,12 @@ const MarketplacePluginInstall = ( {
 			/>
 			{ siteId && <QueryJetpackPlugins siteIds={ [ siteId ] } /> }
 			<Masterbar>
-				<WordPressWordmark className="marketplace-plugin-install__wpcom-wordmark" />
-				<Item>{ translate( 'Plugin Installation' ) }</Item>
+				<div className="marketplace-plugin-install__masterbar">
+					<WordPressWordmark className="marketplace-plugin-install__wpcom-wordmark" />
+					<span className="marketplace-plugin-install__plugin-installation-text">
+						{ translate( 'Plugin installation' ) }
+					</span>
```

Comment:
Can't we reuse the `<Item>` and just override the css rules needed? In terms of placement

```
.masterbar {
    justify-content: flex-start;
}
.masterbar__item {
   flex-grow: 0;
}
```
I think are enough? 

Not sure why Checkout has opted to use a new element rather than reusing `<Item>`. Both interfaces should have the same look and feel.

Reply:
Great point, thanks, I've updated the styling. 👍 

One interesting note: `<Item>` always renders a `button` or an `a` element, so the hover state needs to be overridden in CSS. Maybe that's why Checkout opted to add custom elements.